### PR TITLE
CAMEL-11012 bindy csv doesn't populate with defaultValue on marshal

### PR DESCRIPTION
**Pull request by igarashitm**

### CODE REVIEW

**Comment by oscerd** [comment]
Looks good to me. @davsclaus what do you think?

File: components/camel-bindy/src/test/java/org/apache/camel/dataformat/bindy/csv2/BindyMarshalUnmarshalWithDefaultValueTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.dataformat.bindy.util.ConverterUtils;
+import org.apache.camel.model.dataformat.BindyType;
+import org.apache.camel.test.junit4.CamelTestSupport;
+import org.junit.Test;
+
+/**
+ *
```
Comment (apupier) [inline components/camel-bindy/src/test/java/org/apache/camel/dataformat/bindy/csv2/BindyMarshalUnmarshalWithDefaultValueTest.java]: empty javadoc?

**Comment by davsclaus** [comment]
+1

**Comment by oscerd** [comment]
Thanks @igarashitm the PR has been merged. Do you mind closing this?

**Comment by igarashitm** [comment]
Thanks! closing.

**Pull request closed without merging**